bin/src/command/mod.rs

```diff
@@ -66,60 +68,86 @@ enum CommandMessage {
     WorkerClose {
         id: u32,
     },
+    Advancement {
+        request_identifier: RequestIdentifier,
+        response: Response,
+    },
     MasterStop,
 }
 
-pub enum OrderSuccess {
-    ClientClose(String),
-    ClientNew(String),
-    ClientRequest,
-    DumpState(CommandResponseData), // contains the cloned state
-    LaunchWorker(u32),
-    ListFrontends(CommandResponseData),
+#[derive(Clone)]
```

Comment:
You could add others traits to it like those:

```
#[derive(PartialEq, Eq, PartialOrd, Ord, Clone, Debug)]
```

Reply:
Sure, only `Ord` and `PartialOrd` are not implemented by `CommandResponseData`, which is encapsulated but out of the crate. I'll let those out.